participant/public/data/submissions.not-prod.json

```diff
@@ -26,18 +26,20 @@
         "contact": {
           "phoneNumber": "(406) 987 - 6543",
           "additionalInfo": ""
+        }
+      },
+      "documents": [
+        {
+          "filename": "febpaystubs.pdf",
+          "accepted": true,
+          "s3Url": "https://files.consumerfinance.gov/f/documents/cfpb_building_block_activities_how-to-read-pay-stub_handout.pdf"
```

Comment:
Would be much more robust to have these be full records because this is seed data for the participant. Not just staff

Reply:
Add test files to fixtures folder

Reply:
Done.